docs/source/user_guide/tutorials/getting_started.rst

```diff
@@ -253,484 +133,518 @@ For example, if you ran the ``easylink run`` command on your laptop, the impleme
-   Job 0: Grabbing final output
-   Reason: Missing output files: result.parquet; Input files updated by another job: input_validations/final_validator, intermediate/step_4_python_pandas/result.parquet
-   $ pqprint results/2025_05_06_08_44_38/result.parquet 
-         foo bar  counter  added_column_59  added_column_60  added_column_61  added_column_62  added_column_63
-   0         0   a       63             59.0             60.0             61.0             62.0               63
-   1         1   b       63             59.0             60.0             61.0             62.0               63
-   2         2   c       63             59.0             60.0             61.0             62.0               63
-   3         3   d       63             59.0             60.0             61.0             62.0               63
-   4         4   e       63             59.0             60.0             61.0             62.0               63
-   ...     ...  ..      ...              ...              ...              ...              ...              ...
-   59995  9995   a        1              0.0              0.0              0.0              0.0               63
-   59996  9996   b        1              0.0              0.0              0.0              0.0               63
-   59997  9997   c        1              0.0              0.0              0.0              0.0               63
-   59998  9998   d        1              0.0              0.0              0.0              0.0               63
-   59999  9999   e        1              0.0              0.0              0.0              0.0               63
-   [60000 rows x 8 columns]
-
-As you can see, the output shows that 63 columns were added, as expected.
+              INPUT_DATASET: input_file_ssa
+      - determining_exclusions:
+          implementation:
+            name: default_determining_exclusions
+            configuration:
+              INPUT_DATASET: input_file_w2
+        removing_records:
+          implementation:
+            name: default_removing_records
+            configuration:
+              INPUT_DATASET: input_file_w2
+  clustering:
+    substeps:
+      ...
+  updating_clusters:
+    implementation:
+      name: default_updating_clusters
+
+The last step shown, ``updating_clusters``, looks similar to ``canonicalization_and_downstream_analysis`` above; it simply chooses 
+an implementation for the step using the ``name`` key. 
+
+The substeps of ``clustering`` are hidden -- we'll look at them next. 
+
+The complicated part is ``determining_exclusions_and_removing_records`` and its ``clones`` key:
+
+The schema can define steps as :ref:`cloneable_sections`, which create 
+multiple copies of that section and allow different implementations or inputs to be defined 
+for each copy. We can see that the :ref:`entity_resolution_sub_steps` schema section defines
+``determining_exclusions`` and ``removing_records`` as cloneable in the diagram 
+(blue dashed box).
```

Comment:
I think it might be worth adding a sentence or two here about what the "determining exclusions" and "removing records" steps do. I think even for people familiar with record linkage, it might not be obvious why these are here. I eventually figured out that if I click on the "Entity resolution sub-steps" link above and then scroll down, I can find definitions of these steps. But on a cursory first pass, this is a prominent chunk of the yaml file whose purpose was unclear to me, which was a bit disorienting.